TPP.Core/Commands/Definitions/UserCommands.cs

```diff
@@ -228,5 +239,25 @@ public async Task<CommandResult> Donate(CommandContext context)
                 ResponseTarget = ResponseTarget.Chat
             };
         }
+
+        public async Task<CommandResult> List(CommandContext context)
+        {
+            Role role = await context.ParseArgs<Role>();
+            List<User> users = await _userRepo.FindAllByRole(role);
+
+            return new CommandResult
+            {
+                Response = users.Count > 0 ? $"The users with the '{role.ToString()}' role are: {string.Join(", ", users.Select(u => u.SimpleName))}" : $"There are no users with the '{role.ToString()}' role."
```

Comment:
can you split the ternary into multiple lines for better readability? Same for the ShowRoles command

Reply:
Also please use Name instead of SimpleName for all user-facing texts. logging entries should just rely on User objects' ToString method since that also prints the user ID which can be useful. I don't know of any use cases where outputting the SimpleName is preferable